dev-utils/update-tav-versions.js

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+/** @typedef {import('semver').SemVer} SemVer */
+/**
+ * @typedef {Object} TavConfig
+ * @property {String} [name]
+ * @property {String} versions
+ * @property {String} node
+ * @property {String[]} commands
+ * @property {Object} update-versions
+ * @property {String} update-versions.mode
+ * @property {String} update-versions.range
+ */
+
+'use strict';
+
+// This script scans the `tav.yml` file placed at the root folder and for each
+// entry checks if the `update-versions` property is defined. In that case
+// the script will update the versions property according to the `update-versions`
+// configuration:
+// - range: only versions satisfiying this range will be in the update
```

Comment:
This should be updated to discuss `include`, `exclude` instead now.

Reply:
done!